internal/notify/http.go

```diff
@@ -55,6 +55,10 @@ func (n *Notify) handleNotify(w http.ResponseWriter, r *http.Request) error {
 		return authRequiredError{}
 	}
 
+	w.Header().Set("Content-Type", "application/octet-stream")
+	w.WriteHeader(http.StatusOK)
+	w.(http.Flusher).Flush()
```

Comment:
The status code will be set automatically by line 61 (old) and the flush will be done without blocking in line 68 (old). 

I would set the header before the auth-check so if an anomoyous users gets an 400er, the content-type will still be set.

I already did this in another pull request. Therefore I close this one.